scripts/mkreleaselog

```diff
@@ -126,6 +126,42 @@ indent() {
     sed -e 's/^/  /'
 }
 
+group_changes() {
+    content=$(cat)
+    new_features=$(echo "$content" | grep -i '^- feat:' || true)
+    improvements=$(echo "$content" | grep -i '^- fix:' || true)
```

Comment:
it would be awesome to fit [breaking changes](https://www.conventionalcommits.org/en/v1.0.0/) into this too (feedback from @rjan90 @aarshkshah1992 @ZenGround0 would be good on whether they're comfortable with full CC style):

```suggestion
    new_features=$(echo "$content" | grep -i '^- feat:' || true)
    improvements=$(echo "$content" | grep -i '^- fix:' || true)
    breaking_changes=$(echo "$content" | grep -iE '^- [^:]+!:' || true)
```